libcontainer/userns/userns_linux_fuzzer.go

```diff
@@ -1,5 +1,5 @@
-//go:build gofuzz
-// +build gofuzz
+//go:build linux && gofuzz
+// +build linux,gofuzz
```

Comment:
Needs a rebase as #4329 is now merged

Reply:
Thanks! Just did; also fixed the commit message 👍 🫶